src/common/drpc.c

```diff
@@ -130,16 +130,19 @@ new_unixcomm_socket(int flags)
 
 	D_ALLOC_PTR(comm);
 	if (comm == NULL) {
+		D_ERROR("Failed to allocate unixcomm\n");
 		return NULL;
 	}
 
 	comm->fd = socket(AF_UNIX, SOCK_SEQPACKET, 0);
 	if (comm->fd < 0) {
+		D_ERROR("Failed to open socket\n");
```

Comment:
[Not requesting any changes] I usually also print rc (or errno in this case) if I bother to add a log message like this.

Reply:
It's a good idea - added.